recommender/engine/agents/rl_agent/models/history_embedder.py

```diff
@@ -82,25 +94,25 @@ def __init__(self, SE: int, N=20, layer_sizes=(256, 128), num_layers: int = 3, d
 
         self.network = nn.Sequential(*layers)
 
-    def forward(self, temporal_data: torch.Tensor) -> torch.Tensor:
+    def forward(self, service_history: torch.Tensor) -> torch.Tensor:
         """
-        RNN is used for reducing history's N dimension.
+        Used to process the services history.
```

Comment:
this docstring content should be as similar as possible to docstring in forward method in LSTM History embedder class